### ISSUE TITLE
CSV-Import ignores group field

### ISSUE THREAD

**Issue by radarcapecoder** [keepassxreboot/keepassxc#10614]
## Overview
When importing passwords from a CSV file, the group column seems to be ignored by KeyPassXC.

## Steps to Reproduce
1. Create an CSV file containing password data by, for example, doing an export from Password Gorilla
2. Import CSV file to a new or existing database (doesn't matter)
3. Assign columns from CSV file (group column is here of interest)
4. Import the CSV file

## Expected Behavior
I would expect that the imported password entries would be added to a group folder inside KeePassXC with the same name as the group name in the CSV file. If there is no group folder with the same name, KeePassXC should create a new folder with that name.

Expected behaviour in pseudo code:

if group_column not null then
  if group-folder does not exist then
    create group-folder
  fi
  import CSV row to group-folder
else
  import CSV row to root-folder/"passwords"-folder
fi

## Actual Behavior
All CSV entries will be added to the root-Folder (if new database created) or a folder named "Passwords" (if CSV is imported to an existing database). The group inside the CSV file is being ignored during the import.

## Context
KeePassXC - Version 2.7.7
Revision: 68e2dd8

Qt 5.15.11
Diagnosemodus ist deaktiviert.

Betriebssystem: Windows 10 Version 2009
CPU-Architektur: x86_64
Kernel: winnt 10.0.19045

Aktivierte Erweiterungen:
- Auto-Type
- Browser-Integration
- Passkeys
- SSH-Agent
- KeeShare
- YubiKey
- Schnelle Entsperrung

Kryptographische Bibliotheken:
- Botan 3.1.1
-

**Comment by radarcapecoder** [comment]
Example of the CSV file being used for the import:

```
uuid,group,title,url,user,password,notes
00004f73-c925-1354-71ed-d6330b3b4556,GroupA,UserA,http://www.somedomain.com,NoteA,PasswordA,
00007f42-cb57-4341-2cfc-349e4cccf829,GroupB,UserB,http://www.otherdomain.net,NoteB,PasswordB,
```

**Comment by droidmonkey** [comment]
Fixed for 2.7.8

https://github.com/keepassxreboot/keepassxc/pull/10575

To overcome this for now, just add a / in front of the group names. Or use a snapshot build to do your import: https://snapshot.keepassxc.org
